### DISCUSSION TITLE
Add icons to tab titles in stacking/tabbed view

### DISCUSSION THREAD

**Discussion by l-j-g** [Manjaro-Sway/manjaro-sway#192] [Ideas]
I think it would  look nice to add the workspace icons (sworkstyle?)  to the title bar of stacking / tabbed view -- not sure how difficult this would be to implement? , i'm thinking would probably require a change to sway itself and isn't really a configurable option?

**Comment by boredland** [comment]
I have no idea how one would do that. sworkstyle uses the sway ipc interface, but i have no idea if one should mess with titles...
